src/parser/jobs/smn/modules/MissingDoTs.js

```diff
@@ -0,0 +1,135 @@
+import {t} from '@lingui/macro'
+import {Trans, Plural} from '@lingui/react'
+import React from 'react'
+
+import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import {RotationTable} from 'components/ui/RotationTable'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+// In potency
+const MISSING_DOT_SEVERITY = {
+	1: SEVERITY.MEDIUM,
+	1000: SEVERITY.MAJOR,
+}
+
+// Statuses that need to be up for Fester, Bane, and Ruins to do the most
+const SMN_DOT_STATUSES = [
+	STATUSES.BIO_III.id,
+	STATUSES.MIASMA_III.id,
+]
+
+export default class MissingDoTs extends Module {
+	static handle = 'missingdots'
+	static title = t('smn.missingdots.title')`Missing DoTs`
+	static dependencies = [
+		'enemies',
+		'suggestions',
+		'timeline',
+	]
+
+	_badDotReqCasts = {
+		[ACTIONS.FESTER.id]: {
```

Comment:
would _somewhat_ rather the action -> potency config was up top in a const - then the entire shape of this can be generated at runtime.

doing that would also let you re-use that config for the hook filter in the constructor